owners/src/notifier.js

```diff
@@ -173,9 +173,22 @@ class OwnersNotifier {
    */
   getOwnersToNotify() {
     const notifies = {};
+
+    function notifyOrRequired(subtree, filename) {
+      const notifyOwners = subtree.getModifiedFileOwners(
+        filename,
+        OWNER_MODIFIER.NOTIFY
+      );
+      const requiredOwners = subtree.getModifiedFileOwners(
+        filename,
+        OWNER_MODIFIER.REQUIRE
+      );
+
+      return Array.from(new Set(notifyOwners.concat(requiredOwners)));
```

Comment:
Supernit: `[...notifyOwners, ...requiredOwners]` (I think it reads better)

Reply:
Done